pages/learn/develop/runtime.md

```diff
@@ -249,74 +249,80 @@ At the time of writing there is an inconsistency in the behaviour of `/tmp` dire
 
 ### Mounting external storage media
 
-Mounting external storage media, such as SD cards or USB thumb drives, within your application (running inside Docker) works somewhat different compared to mounting devices directly in Linux. Here we include a set of recommendations that helps you can get started.
+Since the release of multicontainer on the {{ $names.company.lower }} platform we no longer recommend the use of an initsystem in the container. This affects the way we deal with external storage since previously we relied on `systemd`/`OpenRC` and `/etc/fstab`.
 
-**Without the init system**
+The recommended way for mounting external storage media (SD cards, USB sticks, external drives, etc) into a container is now through the use of `mount`. Here we include a set of recommendations that will help you get started.
 
-If you have not enabled an init system in your application or chose to mount manually, you can add the mount logic into your start script. This can be made simpler by adding the storage media settings to `/etc/fstab` in your Dockerfile:
-```Dockerfile
-RUN echo "LABEL=mysdcard /mnt/storage ext4 rw,relatime,discard,data=ordered 0 2" >> /etc/fstab
-```
-Modify your settings as appropriate (device identification, mount endpoint, file system, mount options), and see more information about the possible settings at the [fstab man page](http://man7.org/linux/man-pages/man5/fstab.5.html).
+**{{ $names.os.lower }} kernel support**
 
-Then in your start script you need to create the mount directory and mount the device:
-```Bash
-mkdir -p /mnt/storage && mount /mnt/storage
-```
+Before you start it's a good idea to check if the {{ $names.os.lower }} kernel you are running was compiled with support for the filesystem you want to use. To do so, you can run this command on the **host** which will produce a list of supported filesystems: `cat /proc/filesystems`.
 
-**Using systemd**
+If your filesystem is not supported you can contact us through our [forums](https://forums.balena.io/) and we will be glad to help.
 
-Normally systemd mounts entries from `/etc/fstab` on startup automatically, but running within Docker, it will only mount entries that are not block devices, such as `tmpfs` entries. For non-block devices, adding entries `/etc/fstab` is sufficient, for example in your Dockerfile:
-```Dockerfile
-RUN echo "tmpfs  /cache  tmpfs  rw,size=200M,nosuid,nodev,noexec  0 0" >> /etc/fstab
-```
+__Note:__ You can get a list of device names, labels and filesystem types by running `lsblk -f` (both on the host or container).
```

Comment:
This is entirely optional but maybe this note fits better under the General Tips heading as that's where using the output of this is discussed?

Reply:
Agreed. It was originally there before I removed that section. Now that it's back I should move it.